### DISCUSSION TITLE
How the project fits into current nix-based tooling landscape?

### DISCUSSION THREAD

**Discussion by keinsell** [numtide/devshell#326] [Ideas]
What's the exact goal of the project where tools like `devenv`, `devbox` and similar are gaining popularity and seem to have much higher traction than `devshell` itself - what are differences between `devshell` and `devenv` for example?

*Comment was refactored*

**Comment by adrian-gierakowski** [comment]
This project predates those others by at least 2 years, and I’d hazard a guess that it served as inspiration for their authors. 

Btw I’m not a maintainer, nor a contributor to this project, but I’d like to gently point out that the “tone of voice” of your message comes across as slightly impolite.

**Reply by keinsell** [reply]
> This project predates those others by at least 2 years, and I’d hazard a guess that it served as inspiration for their authors.

I would not deny this, however I still have trouble understanding place for the project in development environment tooling comparing to the potential successors.

> but I’d like to gently point out that the “tone of voice” of your message comes across as slightly impolite.

My bad, sorry.
